frontend/src/modules/EditZing/Components/StudentGrid.tsx

```diff
@@ -1,37 +1,44 @@
 import React from 'react'
-import { makeStyles, createStyles, Theme } from '@material-ui/core/styles'
-import Paper from '@material-ui/core/Paper'
-import Grid from '@material-ui/core/Grid'
+import { styled } from '@mui/material/styles'
+import Paper from '@mui/material/Paper'
+import Grid from '@mui/material/Grid'
 import { STUDENT_TYPE } from 'EditZing/Types/Student'
 import { StudentGridProps } from 'EditZing/Types/ComponentProps'
-import { genderSVG } from 'EditZing/Styles/InlineSVGs'
+// import { genderSVG } from 'EditZing/Styles/InlineSVGs'
 import { colors } from '@core'
 import { useDrag } from 'react-dnd'
 import { StyledStudentDetail } from 'EditZing/Styles/StudentAndGroup.style'
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      flexGrow: 1,
-    },
-    paper1: {
-      padding: theme.spacing(2),
-      textAlign: 'left',
-      color: colors.black,
-      fontFamily: 'Montserrat',
-      fontWeight: 700,
-      fontSize: 14,
-      border: '0px solid rgba(205, 156, 242, 0.15)',
-      boxShadow: '0px 2px 5px rgba(205, 156, 242, 0.2);',
-      borderRadius: '10px',
-    },
-    paper2: {
-      display: 'flex',
-      flexDirection: 'column',
-      gap: '2px',
-      marginTop: '2px',
-    },
-  })
+const PREFIX = 'StudentGrid'
+
+const classes = {
+  paper1: `${PREFIX}-paper1`,
+  paper2: `${PREFIX}-paper2`,
+}
+
+// should we make it so that when it overflows, then the user can scroll?
```

Comment:
Scrolling in a little box like this might look weird, not sure what a better solution is though

Reply:
Hmm, how should we fix this then? Also I'm not sure how it looks in the non upgraded version, because it seems like there is even less space after upgrading causing overflow. It also might be due to the fact that I'm on a Macbook Air currently so everything looks different.

Reply:
We should prob talk with design about it at some point, might involve using different breakpoints for the unmatched/group grids or using a min-width for the student card. Or maybe hiding overflow and showing all the details in a tooltip

Reply:
Yeah makes sense especially with the breakpoints, which hopefully should also be more fluid with the new component library, so should I just push what we got here for now?

Reply:
Yes